data_structures/queue/queue.py

```diff
@@ -0,0 +1,50 @@
+from node import Node
+
+
+class Queue:
+    """Create queue object."""
+    def __init__(self, iter=[]):
+        self.front = None
+        self.back = None
+        self._size = 0
+        try:
+            for item in iter:
+                self.enqueue(item)
+        except ValueError:
+            print('Value error! Please insert an iterable.')
```

Comment:
Instead of printing an error, it is better practice in this case to actually raise an exception.